dask/array/linalg.py

```diff
@@ -33,7 +33,7 @@ def _nanmin(m, n):
     return k_1 if np.isnan(k_0) else k_0
 
 
-def tsqr(data, name=None, compute_svd=False, _max_vchunk_size=None):
+def tsqr(data, compute_svd=False, _max_vchunk_size=None):
```

Comment:
This one makes me a little nervous as it is in the middle of the arguments and could cause issues if people are relying on positional arguments to fill these somehow.